controllers/workplacement_controller_test.go

```diff
@@ -165,7 +165,7 @@ var _ = Describe("WorkplacementReconciler", func() {
 				Expect(result).To(Equal(ctrl.Result{}))
 
 				By("calling UpdateFiles()")
-				Expect(fakeWriter.UpdateFilesCallCount()).To(Equal(1))
+				Expect(fakeWriter.UpdateFilesCallCount()).To(Equal(2))
```

Comment:
Why is this needed?

Reply:
the reconciliation loop runs twice, because the finalizer is now after the update an additional call is made to the update. Before it was finazlers->requque->finalizers->update->done, now its update->requeue->update->finalizers->done